jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/MavenWebAppContext.java

```diff
@@ -231,7 +231,7 @@ public void setResourceBases(String[] resourceBases)
             // This is a user provided list of configurations.
             // We have to assume that mounting can happen.
             List<URI> uris = Stream.of(resourceBases)
-                .map(URI::create)
+                .map(URIUtil::toURI)
```

Comment:
ResourceFactory is the class we should be using to convert user provided strings. Especially as what we want is a CompositeResource at the end.